recipes/opencv/4.x/conanfile.py

```diff
@@ -140,6 +148,8 @@ def requirements(self):
             self.requires("eigen/3.3.9")
         if self.options.parallel == "tbb":
             self.requires("tbb/2020.3")
+        if self.options.with_ipp and self.options.ipp_package:
+            self.requires(str(self.options.ipp_package))
```

Comment:
🟨Defiantly will require a CCI team member to review... I've not see this before 😱

Reply:
Well, I need to use a private Conan IPP package internally (IPP is licensed by my company from Intel so I cannot make it public). It's done with `-o opencv:with_ipp=True -o opencv:ipp_package=ipp/2018.3@my_company/stable`. But others can use the special IPP version for OpenCV leaving `ipp_package` option set to `None`

Reply:
Ah, looks like IPP is now a part of Intel oneAPI and a commercial license is no longer needed. Well, then OpenCV recipe could use oneAPI Conan package (alas, not existing yet)

Reply:
IPP even in oneAPI is not an easy nut to handle - it requires registration on Intel web-site to download, and also it provides a GUI installer. need to check if there is an easy way to obtain it.

Reply:
@SSE4  
> IPP even in oneAPI is not an easy nut to handle - it requires registration on Intel web-site to download, and also it provides a GUI installer. need to check if there is an easy way to obtain it.

I thought IPP sources were available from Github just as for other oneAPI libraries. But looks like it's not the case (although IPP sources are available from Intel with registration). When why not package IPP binaries that are already available for OpenCV - the approach that I initially proposed in this PR?

Reply:
@db4 check this PR: https://github.com/conan-io/conan-center-index/pull/7477
it explains why can't we package IPP in ConanCenter